### ISSUE TITLE
 Incorrect vulnerability details: postcss@8.2.13

### ISSUE THREAD

**Issue by EzeCuervo** [OSSIndex/vulns#194]
**Vulnerability URL**
```
https://ossindex.sonatype.org/component/pkg:npm/postcss@8.2.13
```

**Description**
PostCSS fixed the ReDoS known vulnerability in 8.2.13 version.
```
https://github.com/postcss/postcss/releases/tag/8.2.13
```

Thanks!!

**Comment by ken-duck** [comment]
Thanks for the heads up. It should be fixed in the public DB by sometime tomorrow.

**Comment by ndonewar** [comment]
Closing this issue as it appears to be resolved and version 8.2.13 is no longer included. Please feel free to reopen or create a new issue if needed. Thanks!
